### PR TITLE
Advance to year field when / is pressed

### LINKED ISSUE #36: slash key should move input to year field
When filling out an expiration date it would be nice if typing `1/16` would work just like `1` <kbd>Tab</kbd> `16`

### PR DESCRIPTION
**Pull request by samuelli**
Fixes #36

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -27,7 +27,8 @@
     "iron-form-element-behavior": "PolymerElements/iron-form-element-behavior#^1.0.0",
     "paper-input": "PolymerElements/paper-input#^1.0.0",
     "paper-styles": "PolymerElements/paper-styles#^1.0.0",
-    "polymer": "Polymer/polymer#^1.1.0"
+    "polymer": "Polymer/polymer#^1.1.0",
```
Comment (notwaldorf) [inline bower.json]: alphabetical order, please!

**Comment by notwaldorf** [comment]
Would you mind adding a test for this, please? Check out `MockInteractions` in `iron-demo-helpers` for key events help 😊

File: bower.json (hunk 1 above)
Comment (samuelli) [inline bower.json]: Done.

**Comment by samuelli** [comment]
Having a lot of trouble with this. The following works for `a` but not for `@` and `/`.

`keyBindings: {
      'a': '_selectYear',
      '@': '_selectYear',
      '/': '_selectYear'
},`

so I'm not sure if iron-a11y-keys-behavior is broken. There's a test for `@` but I haven't been able to get it work outside of the test.

**Comment by notwaldorf** [comment]
paging @valdrinkoshi, our `a11y-keys` keeper :)

**Comment by valdrinkoshi** [comment]
Hi @samuelli, I'm not sure I understand where's the problem. Is it in the tests?
Since the three bindings point to the same method, you can just do:
```javascript
keyBindings: { 'a @ /' : '_selectYear' }
```
http://jsbin.com/ciboye/1/edit?html,output

**Comment by samuelli** [comment]
@valdrinkoshi: Only the `a` key seems to work in that demo. I'm unable to trigger on `@` or `/`.

**Comment by samuelli** [comment]
Those keys seem to work now. Added a test but its blocked on https://github.com/PolymerElements/iron-test-helpers/pull/47/files

**Comment by samuelli** [comment]
ping @notwaldorf

**Comment by notwaldorf** [comment]
Thanks! LGTM

**Pull request merged**